### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by MMikhailova** [lab-brussels-1/The-first-group#29] [General]
Review

**Comment by sammou00** [comment]
![image](https://user-images.githubusercontent.com/67711948/159561711-e495ac3c-1b41-4da0-9125-e0d5c6944d2b.png)

**Comment by sammou00** [comment]
Hello, well done , some notes maybe help you to make it even better :

for `Plantsman` persona I think it is quite clear and nicely presented .

for the first persona is it `middle-school student` ? if so then remove the text `knowledge about tress` and put `middle-school student` at it place.

for example : 
middle school student this my user persona , which means he will come and see my website about trees 
**Bio**  Sara is a middle school student , she is  writing a short story about trees .
**goals** : she need more simple information about trees that 12 years old student can understand 
**Frustrations** : what is Sara's need or goal to visit my website ? well there are a lot of scientific website about trees , but Sara is only 12 , she find it very difficult to understand these complected scientific websites so she will come to your website  because it has information she can understand.
**Scenario** : Sara's teacher asked her to write a story about trees

**Reply by MMikhailova** [reply]
Hello @Samir

thank you for your comment and I agree about Plantsman persona:)
Middle school student is a persona created in template on https://www.semrush.com. You will see its description if you click the link (Middle school student). 

Knowledge about tress is another persona.

**Comment by sammou00** [comment]
Hello, Ok, but knowledge about tress is not  a person ? 

maybe environmental activist , which is a persons or green protector will be better than knowledge about tress.

second : if middle -school student is another persona then maybe add more info about it same like other personas , and add link too for more clarity.

**Reply by MMikhailova** [reply]
Thank you for the good idea that we will apply! :)

**Comment by sammou00** [comment]
`Review` : 
You need to understand that each developer do things little bite differently , it is like each developer has his /her styles.
My Review : 
1- First well done. You have really done a very good job. We all are proud of you .
2 - Second : 
Why do we need `user-personas` ? 
 because it gives us an idea who will use this website.
for example:  `GitHub` is a server or website target programmer , other people can use it but it design for programmer . User-persona is `The public`  is very general. 

3- For your design it is really good small thing . we use 
![image](https://user-images.githubusercontent.com/67711948/159979146-1cb7956c-e114-47a7-bb97-30b4b2934361.png)
 to show that is it is a picture .

4- I see very nice comments in your `index.html` but no comments in your `css`

5- Your development-strategy is really nice , just one small thing
![image](https://user-images.githubusercontent.com/67711948/159980030-75e946b2-2f32-4d23-b292-f44a190b8e67.png)
Why (milestone) ? each feature goes to one branch you can use Navbar (new branch).
Milestone : normally related to time like what should we have finished in , week1 , week2 , week3

**Reply by minjupgeorge** [reply]
@samirm00 
Thank you for your valuable comments.

1. We will add comments to the css file.
2. As you said "The public " in User persona is a general one . Need to update it
3. I understood clearly about the milestone now. at initial stage I had a confusion regarding  milestones and issues. I will remove milestones from development strategy.
4. The picture in the design.md also needs to be represent as
![image](https://user-images.githubusercontent.com/32439811/160028054-e182f2fb-2500-4013-bfe3-04afb3aabec3.png)
